packages/backend/src/users/users.handlers.ts

```diff
@@ -68,6 +68,35 @@ export function getUserById(userId: string): Promise<null | User> {
           as: '_solutions',
         },
       },
+      {
+        $lookup: {
+          from: 'tasks',
+          let: {
+            tasks: '$_tasks',
+          },
+          pipeline: [
+            {
+              $match: {
+                $expr: {
+                  $in: ['$_id', '$$tasks'],
+                },
+              },
+            },
+            {
+              $lookup: {
+                from: 'users',
+                localField: '_user',
+                foreignField: '_id',
+                as: '_user',
+              },
+            },
+            {
+              $unwind: '$_user',
+            },
+          ],
+          as: '_tasks',
```

Comment:
rozumiem że ten handler zwróci odpowiedź z listą tasków dla pola `_tasks` - a umawialiśmy się, żeby stałe z `_` zawierały tylko listy ID-ków,
więc jeśli jest tak jak piszę, to zmień proszę z `_tasks` na `tasks` i wyżej z `_solutions` na `solutions`

Reply:
zmienione